bff/src/Duende.Bff.Blazor.Client/ClaimRecord.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Text.Json.Serialization;
+
+namespace Duende.Bff;
+
+/// <summary>
+/// Serialization friendly claim.
+///
+/// Note, this is a copy of the ClaimRecord class from Duende.Bff, but since we can't create a reference to it, we need to copy it here.
+/// We also can't link to it (as we do with the extensions) because we had to make it public. 
+/// </summary>
```

Comment:
Comment seems out of date, since this isn't public anymore. Does that mean we can link to a shared file?

Reply:
as discussed, i changed the comment.